startblock.gemspec

```diff
@@ -27,7 +27,10 @@ start on a working app.
   s.version = Startblock::VERSION
 
   s.add_development_dependency "minitest", "~> 5.8"
+  s.add_development_dependency 'rails', Startblock::RAILS_VERSION
+  s.add_development_dependency 'railties', Startblock::RAILS_VERSION
   s.add_runtime_dependency 'bundler', '~> 1.3'
   s.add_runtime_dependency 'rails', Startblock::RAILS_VERSION
+  s.add_runtime_dependency 'railties', Startblock::RAILS_VERSION
```

Comment:
Right, I'll remove this because I think this only fixed a symptom with the bundler stuff